### ISSUE TITLE
Loss function from default_config.yml not overwritten by new config

### ISSUE THREAD

**Issue by saibr** [ecmwf/WeatherGenerator#2440]
### What happened?

When training a model with a custom loss function, e.g. kernel_crps, my model still uses the mse loss as a second loss. It seems that OmegaConf is merging the losses dict from the custom_config file with the dict from default_config.yml, instead of overwriting it.

I fixed this by specifying "mse": null in my custom_config and added `if params is not None` in the following part in loss_module_physical.py:
```
self.loss_fcts = [
            [
                getattr(loss_fns, name),
                params.get("weight", 1.0),
                name,
            ]
            for name, params in loss_fcts.items()
            if params is not None
        ]
```
